utility/src/main/java/org/oppia/android/util/parser/image/UrlImageParser.kt

```diff
@@ -460,17 +463,17 @@ private fun calculateInitialMargin(availableAreaWidth: Int, drawableWidth: Float
   return margin.coerceAtLeast(0f)
 }
 
-// Reference: https://stackoverflow.com/a/51865494
+// Reference: https://stackoverflow.com/a/57495622/12314934
 private fun TextView.width(computeWidthOnGlobalLayout: (Int) -> Unit) {
-  if (width == 0) {
+  if (isLaidOut) {
```

Comment:
Invert this condition (so again the difference is minimized between the two to better emphasize the change).

Reply:
Now, we don't need this.